.github/workflows/docker_publish.yml

```diff
@@ -77,28 +95,28 @@ jobs:
 
     strategy:
       matrix:
-        ubuntu_versions : [
+        ubuntu_version : [
           20.04,
           22.04,
         ]
         compiler : [
           gcc,
           clang,
         ]
-        hdf5_versions : [
+        hdf5_version : [
           1.14.3,
         ]
-        moab_versions : [
+        moab_version : [
           5.4.1,
           5.5.1,
         ]
-        double_down : [
-          OFF,
-        ]
-        geant_version : [
+        geant4_version : [
           10.7.4,
           11.1.2
         ]
+        double_down_version : [
+          off
```

Comment:
This matrix should probably match the matrix above to make sure all the right images exist.